app/experimenter/experiments/models.py

```diff
@@ -377,17 +381,38 @@ def has_normandy_info(self):
         return self.normandy_slug or self.normandy_id
 
     @property
-    def normandy_api_recipe_url(self):
-        if self.normandy_id:
-            return settings.NORMANDY_API_RECIPE_URL.format(id=self.normandy_id)
+    def format_dc_normandy_urls(self):
+        # returns a list of dictionaries containing D.C. and Normandy
+        # urls for the main normandy id and other normandy ids if they exist
+        normandy_recipe_url = settings.NORMANDY_API_RECIPE_URL
+        delivery_console_url = settings.DELIVERY_CONSOLE_RECIPE_URL
 
-    @property
-    def delivery_console_recipe_url(self):
         if self.normandy_id:
-            return settings.DELIVERY_CONSOLE_RECIPE_URL.format(
-                id=self.normandy_id
+            urls = []
+            urls.append(
+                {
+                    "id": self.normandy_id,
+                    "normandy_url": normandy_recipe_url.format(
+                        id=self.normandy_id
+                    ),
+                    "DC_url": delivery_console_url.format(id=self.normandy_id),
+                }
```

Comment:
again these keys have pretty short names... because of line length problems..

Reply:
Yeah that's fine, linters are your friend, do what they say. 👬